build.sbt

```diff
@@ -35,3 +39,11 @@ lazy val `monster-xml-to-json-list` = project
       "org.scalatest" %% "scalatest" % scalaTestVersion
     ).map(_ % Test)
   )
+
+  lazy val `xml-to-json-list-template` = project
+    .in(file("argo"))
```

Comment:
Is "argo" sufficient? Should I mention "chart" or "helm" in the name anywhere?

Reply:
I think argo is sufficient